code/game/objects/items/grenades/ghettobomb.dm

```diff
@@ -159,3 +139,80 @@
 	update_mob()
 	explosion(src.loc,-1,-1,2, flame_range = 4)	// small explosion, plus a very large fireball.
 	qdel(src)
+
+
+// Molotov Cocktail
+/obj/item/reagent_containers/food/drinks/bottle/molotov
+	name = "molotov cocktail"
+	desc = "A empty bottle with a rag in it. Needs to be filled with a flammable liquid before being lit on fire and chucked at someone you don't like."
+	icon = 'icons/fallout/objects/guns/explosives.dmi'
+	lefthand_file = 'icons/fallout/onmob/weapons/special_lefthand.dmi'
+	righthand_file = 'icons/fallout/onmob/weapons/special_righthand.dmi'
+	icon_state = "molotov"
+	item_state = "ied"
+	list_reagents = list()
+	var/list/accelerants = list(	/datum/reagent/consumable/ethanol, /datum/reagent/fuel, /datum/reagent/clf3, /datum/reagent/phlogiston,
+							/datum/reagent/napalm, /datum/reagent/hellwater, /datum/reagent/toxin/plasma, /datum/reagent/toxin/spore_burning)
```

Comment:
```suggestion
	var/list/accelerants = list(
		/datum/reagent/consumable/ethanol,
		/datum/reagent/fuel,
		/datum/reagent/clf3,
		/datum/reagent/phlogiston,
		/datum/reagent/napalm,
		/datum/reagent/hellwater,
		/datum/reagent/toxin/plasma,
		/datum/reagent/toxin/spore_burning,
		)
```
* Properly format lists.
* Use trailing commas.

Some of these reagents are pretty hardcore. Sure you want them around?